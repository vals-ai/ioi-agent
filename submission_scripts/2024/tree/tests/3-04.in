version https://git-lfs.github.com/spec/v1
oid sha256:326514eb20e553d0f7df3f4908fe399e1df82cc834d2f3f05cdca1176d9ba390
size 762237
