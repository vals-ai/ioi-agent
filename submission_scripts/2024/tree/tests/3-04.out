version https://git-lfs.github.com/spec/v1
oid sha256:3b23cd9282d9f2373b1062eccb7ed9f676ee6e83ec59a2f4d405d3677c1a71d5
size 166
